webpack.config.babel.js

```diff
@@ -1,18 +1,19 @@
-const path = require('path');
+const Path = require('path');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const StyleLintPlugin = require('stylelint-webpack-plugin');
-const VisualizerPlugin = require('webpack-visualizer-plugin');
+const TerserPlugin = require('terser-webpack-plugin');
+const VisualizerPlugin = require('webpack-visualizer-plugin2');
 
-const MAX_DEVELOPMENT_OUTPUT_SIZE = 3000000;
-const MAX_PRODUCTION_OUTPUT_SIZE = 335000;
+const MAX_DEVELOPMENT_OUTPUT_SIZE = 3300000;
+const MAX_PRODUCTION_OUTPUT_SIZE = 410000;
```

Comment:
Is it larger because of the exported CSS ⬆️ ?

Reply:
I looked into it and I think the way we use these numebrs makes them a case of a "housenumero".

I created: https://github.com/react-ui-org/react-ui/issues/470